src/Views/HardwareView.vala

```diff
@@ -289,6 +298,18 @@ public class About.HardwareView : Gtk.Grid {
         return _("Unknown Graphics");
     }
 
+    private async void get_graphics_info () {
+        var primary_gpu = yield get_gpu_info (true);
+        primary_graphics_info.label = primary_gpu;
+
+        var secondary_gpu = yield get_gpu_info (false);
+        if (secondary_gpu != null) {
+            secondary_graphics_info.label = secondary_gpu;
+            graphics_grid.add (secondary_graphics_info);
```

Comment:
Another way to do this without adding an additional grid would be to use https://valadoc.org/gtk+-3.0/Gtk.Grid.attach_next_to.html

Reply:
Hm, I tried that, but it doesn't seem to update the items below it in the grid, so you just end up with two labels on top of each other.

Reply:
@danrabbit Here's what I get. The second GPU label correctly attaches below the first, but doesn't push down the subsequent RAM and storage labels:

![Screenshot from 2021-01-27 17-29-05](https://user-images.githubusercontent.com/3372394/106029665-37726600-60c5-11eb-87d1-64621b3a03cd.png)

Reply:
Oh you're right. That's lame. Nevermind then!